.github/workflows/continuous-integration.yml

```diff
@@ -240,11 +242,16 @@ jobs:
           - php-version: "8.5"
             postgres-version: "17"
             extension: "pdo_pgsql"
+          - php-version: "8.4"
+            postgres-version: "15"
+            extension: "pdo_pgsql"
+            postgres-initdb-args: "--locale-provider=icu --icu-locale=en-US"
```

Comment:
Could this be replaced with something like:
```yaml
locale-provider: icu
```

Then, based on the value of `locale-provider` from the matrix, the actual arguments could be constructed, similar to this: https://github.com/doctrine/dbal/blob/0a6c3c2ef017a49e4e411a314e53d911e7bff2fe/.github/workflows/continuous-integration.yml#L114-L115

It would help keeping the job names shorter and avoid spaces and potentially other special characters in the coverage file name.

Reply:
It's a bit clumsy but I tried to adapt it.